saagieapi/pipelines/pipelines.py

```diff
@@ -161,35 +160,6 @@ def get_instance(self, pipeline_instance_id: str, pprint_result: Optional[bool]
             pprint_result=pprint_result,
         )
 
-    @deprecation.deprecated(
```

Comment:
Do you plan to do a major release that brings breaking changes? If not, users will be affected by the removal of these deprecated methods and won't be warned...

Reply:
Major release planned